Error occurred during training. Error: name 'MODEL_ARTIFACTS_PATH' is not defined
Traceback (most recent call last):
  File "train.py", line 98, in run_training
    default_hyperparameters = run_hyperparameter_tuning(X_train,Y_train)
  File "/Users/ammar/Ready Tensor/src/hyperparameter_tuning/tuner.py", line 61, in run_hyperparameter_tuning
    joblib.dump(best_hyperparameters,MODEL_ARTIFACTS_PATH+"/optimized_hyper_parameters.joblib")
NameError: name 'MODEL_ARTIFACTS_PATH' is not defined
